### ISSUE TITLE
Hi, I am a beginner 

### ISSUE THREAD

**Issue by Rayren0502** [eriklindernoren/Keras-GAN#14]
Can you please explain why d_loss[0] is the discriminator loss while d_loss[1] is the accuracy?

**Comment by eriklindernoren** [comment]
Hi,

Please read the Keras documentation for questions like these. 

Good luck,
Erik

**Comment by CreeperGo** [comment]
https://keras.io/models/model/

**Comment by CreeperGo** [comment]
Returns

A History object. Its History.history attribute is a record of training loss values and metrics values at successive epochs, as well as validation loss values and validation metrics values (if applicable).
